cmd/cluster-ingress-operator/main.go

```diff
@@ -31,7 +31,7 @@ func main() {
 	if err != nil {
 		logrus.Fatalf("Failed to get watch namespace: %v", err)
 	}
-	resyncPeriod := 5 * time.Second
+	resyncPeriod := 10 * time.Minute
```

Comment:
I don't think this will work at the current operator-sdk version because `resyncPeriod` unit is expected to be seconds... https://play.golang.org/p/4BKjIShbuYm

Might need to change this to `resyncPeriod := 600` until you update to the latest sdk which accepts a `time.Duration`.

Reply:
Disregard, thought I was looking at the cluster-dns-operator repo which is still on the previous version...